### PR TITLE
refactor(server): get `PORT` from env var

### PR DESCRIPTION
**Pull request by ramfox**
Also adding more potentially "reservedUsernames" that may conflict with
app routes

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ var server = app.listen(PORT, () => {
 // are formed
 // must be updated if routes change, particularly routes with 2 segments as
 // those are the routes at risk for being incorrectly matched
-const reservedUsernames = ['ipfs', 'login', 'workflow'] 
+const reservedUsernames = ['ipfs', 'login', 'workflow', 'admin', 'settings', 'signin', 'signout', 'signup', 'userimage', 'search']
```
Comment (Arqu) [inline server/index.js]: `oauth`, `identity`, `client`, `issues`, `follow`, `unpublish`

**Review by Arqu — CHANGES_REQUESTED** [review]
Can we add these too, once ready, I can merge once I settle things in CI

File: server/index.js (hunk 1 above)
Comment (ramfox) [reply server/index.js]: ahhh yes. Sorry I don't want you to this I ignored you when you linked to the backend routes, I must have misunderstood. I didn't think that the react app would ever request these routes. added them!

**Review by Arqu — APPROVED** [review]
🚀

**Pull request merged**